elements/pfe-card/demo/custom-theme.css

```diff
@@ -0,0 +1,233 @@
+  --pfe-theme--zindex--modal: 1001;
+  --pfe-theme--zindex--navigation: 103;
+  --pfe-theme--zindex--subnavigation: 95;
+  --pfe-theme--zindex--content: 0;
+  --pfe-theme--font-size--xxl: 48px;
+  --pfe-theme--font-size--xl: 40px;
+  --pfe-theme--font-size--alpha: 36px;
+  --pfe-theme--font-size--beta: 28px;
+  --pfe-theme--font-size--gamma: 24px;
+  --pfe-theme--font-size--delta: 20px;
+  --pfe-theme--font-size--epsilon: 18px;
+  --pfe-theme--font-size--zeta: 16px;
+  --pfe-theme--font-size: 18px;
+  --pfe-theme--line-height--xxl: 1.2;
+  --pfe-theme--line-height--xl: 1.2;
+  --pfe-theme--line-height--alpha: 1.3;
+  --pfe-theme--line-height--beta: 1.3;
+  --pfe-theme--line-height--gamma: 1.3;
+  --pfe-theme--line-height--delta: 1.5;
+  --pfe-theme--line-height--epsilon: 1.5;
+  --pfe-theme--line-height--zeta: 1.5;
+  --pfe-theme--line-height: 1.5;
+  --pfe-theme--font-weight--xxl: 300;
+  --pfe-theme--font-weight--xl: 300;
+  --pfe-theme--font-weight--alpha: 400;
+  --pfe-theme--font-weight--beta: 400;
+  --pfe-theme--font-weight--gamma: 400;
+  --pfe-theme--font-weight--delta: 400;
+  --pfe-theme--font-weight--epsilon: 400;
+  --pfe-theme--font-weight--zeta: 400;
+  --pfe-theme--font-weight: 400;
+  --pfe-theme--font-family: "RedHatText", "Overpass", Overpass, Helvetica, Arial, sans-serif;
+  --pfe-theme--font-family--heading: "RedHatDisplay", "Overpass", Overpass, Helvetica, Arial, sans-serif;
+  --pfe-theme--font-family--code: "Overpass Mono", Consolas, Monaco, Andale Mono, monospace;
+  --pfe-theme--color--text: #151515;
+  --pfe-theme--color--text--on-dark: #fff;
+  --pfe-theme--color--text--on-saturated: #eee;
+  --pfe-theme--color--link: #0066cc;
+  --pfe-theme--color--link--hover: #004080;
+  --pfe-theme--color--link--focus: #004080;
+  --pfe-theme--color--link--visited: #0066cc;
+  --pfe-theme--color--link--on-dark: #73bcf7;
+  --pfe-theme--color--link--hover--on-dark: #2b9af3;
+  --pfe-theme--color--link--focus--on-dark: #2b9af3;
+  --pfe-theme--color--link--visited--on-dark: #73bcf7;
+  --pfe-theme--color--link--on-saturated: #ffffff;
+  --pfe-theme--color--link--hover--on-saturated: #d2d3d5;
+  --pfe-theme--color--link--focus--on-saturated: #d2d3d5;
+  --pfe-theme--color--link--visited--on-saturated: #ffffff;
+  --pfe-theme--link-decoration: none;
+  --pfe-theme--link-decoration--hover: underline;
+  --pfe-theme--link-decoration--focus: underline;
+  --pfe-theme--link-decoration--visited: none;
+  --pfe-theme--link-decoration--on-dark: none;
+  --pfe-theme--link-decoration--hover--on-dark: underline;
+  --pfe-theme--link-decoration--focus--on-dark: underline;
+  --pfe-theme--link-decoration--visited--on-dark: none;
+  --pfe-theme--link-decoration--on-saturated: underline;
+  --pfe-theme--link-decoration--hover--on-saturated: underline;
+  --pfe-theme--link-decoration--focus--on-saturated: underline;
+  --pfe-theme--link-decoration--visited--on-saturated: underline; }
+
+
+
+.pfe-headline-xxl {
```

Comment:
These classes don't seem necessary here though they could be valuable eventually in the pfe-styles folder as a helper file.

Reply:
I see this is resolved but I didn't see a reply on the thread, did it not go through?

Reply:
Sorry! I pushed up changes and never responded. This file was only for testing so we're deleting it now.